contents/docs/web-analytics/faq.mdx

```diff
@@ -92,11 +92,47 @@ If you track multiple domains, you might need to set up [cross-domain tracking](
 ### Session timeout
 
 1. A user visits a site (for example, `google.com`) that links to yours.
+
 2. The user clicks a link to your site. At this point, a new session starts and the referrer is `google.com`.
+
 3. The user walks away from their computer, and is inactive long enough for the session to expire (30 minutes by default).
+
 4. User comes back to their computer, goes to the already-open browser tab with your site, and clicks an internal link. A new session starts and the referrer is your own site.
 
 If your users are typically inactive for longer than 30 minutes within one period of activity, you can increase the `session_idle_timeout_seconds` setting in your [posthog-js configuration](/docs/libraries/js#config).
 
+## Why does PostHog impact my PageSpeed score?
+
+The simple answer: PostHog consumes resources (CPU) and execution time to load the resources that make it work. This isn't without cost, but PageSpeed Insights makes this cost seem larger than it actually is. 
```

Comment:
Things to add here:
- The only way to guarantee a good PageSpeed score is to run no analytics at all / very basic analytics that provides minimal insight.